lib/puppet/provider/grafana_dashboard_permission/grafana.rb

```diff
@@ -218,7 +218,7 @@ def existing_permissions
     end
   end
 
-  def permission_data(destroy = false) # rubocop:disable Style/OptionalBooleanParameter
+  def permission_data(destroy = false)
```

Comment:
Thoughts for this one?
I added `# rubocop:disable Style/OptionalBooleanParameter`